Controllers/sessions.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcrypt');
+const User = require('../Models/userModel')
+
+router.get('/sessions', (req, res) => {
```

Comment:
Just curious what this is doing?

Reply:
I took that out in the last commit

Reply:
Did you do it on this PR because it shows that it is still here. If you deleted it it would exist in the PR

Reply:
There are two PRs